src/main/java/wgslplugin/language/psi/WGSLElementFactory.java

```diff
@@ -7,10 +7,11 @@
 
 public class WGSLElementFactory {
     public static WGSLVariableIdentDecl createVariable(Project project, String name) {
-        final WGSLFile file = createFile(project, "var " + name + ";");
-        PsiElement[] idents = PsiTreeUtil.collectElements(file, e -> e instanceof WGSLVariableIdentDecl);
+        final WGSLFile file = createFile(project, "var " + name + ": dummy;");
```

Comment:
new set of parsing rules gives an error `WGSLTokenType.COLON, WGSLTokenType.EQUAL or WGSLTokenType.SEMICOLON expected, got ';'` for `var $name;` - this was leading to an error of `PsiElement[] idents` with the size of 0